integration/all/list_containers_test.go

```diff
@@ -93,6 +92,10 @@ func newListContainerTestStep(
 }
 
 func TestListContainers(t *testing.T) {
+	if DefaultTestComponent == InspektorGadgetTestComponent {
+		t.Skip("Skip running list-containers with kubectl-gadget")
+	}
+
```

Comment:
I think it's safer to make this check the other way around:

```go
	if DefaultTestComponent != IgTestComponent {
		t.Skip("Skip running list-containers without ig-gadget")
	}
```

The reason is that we could add more values to DefaultTestComponent. 
Same for other places.